test/controllers/responses_controller_test.rb

```diff
@@ -1,7 +1,73 @@
 require 'test_helper'
 
 class ResponsesControllerTest < ActionDispatch::IntegrationTest
-  # test "the truth" do
-  #   assert true
-  # end
+  include Devise::Test::IntegrationHelpers
+
+  setup do
+    @user  = create(:user)
+    @admin = create(:admin)
+  end
+
+  test "response can only be destroyed by a logged in user" do
+    @survey   = create(:survey, user: @user)
```

Comment:
Every test seems to use @survey, what could we do with it to improve the DRYness?

Reply:
Good idea @spencerldixon !